<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeachingMenu</class>
 <widget class="QWidget" name="TeachingMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="textBubbleLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>450</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="einsteinLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>302</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Wingdings 3</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
